app/controllers/student/justifications_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Student::JustificationsController < StudentController
+  def index
+    @justifications = Justification.all.where(email: current_user.email)
+  end
```

Comment:
@davithss you can create a method on Student model called "justifications" and call here

```ruby
class Student

...

  def justifications
     Justification.all.where(email: self.email)
  end

...
end
```

```ruby
  def index
    @justifications = current_user.justifications
  end
end
```

Reply:
great suggestion! tks!